App.js

```diff
@@ -21,22 +21,22 @@ const {
  * @param {boolean} options.rejectUnauthorized - Should the httpsAgent reject
  *   invalid SSL certificates?
  * @param {string} options.veresMode - The mode for the veres one driver.
- * @param {string} [options.maintainerKey] - A path to a key file for the
- *   primary node.
+ * @param {string} [options.maintainerKeyWord = 'password'] - A secret for the
```

Comment:
Can we change this to use `seed` language and pass a seed in directly like we do with other APIs? And remove the default? Why is there a default? If there needs to be one, we should change the default from `password` to something sufficiently random and large -- but again, why is there one?

Also, why is this using passwords to begin with? This documentation would encourage bad security practices and the code below seems to use them (sha256 hashing of a simple password to use as a key).

We should fix these things or better explain what's going on and why it's ok.

Reply:
What format is the seed? A string? As for using seed instead of `maintainerKeyWord` the issue is with the cli and the cli options. `-s` is currently set to `--secondary` which is used to pass in a comma separated list of secondary nodes. We could use `-m` for `--seedMaterial` however. If we aren't using passwords what is the recommend way of passing seed material to the binary? You can't just pass in a `Uint8Array` raw from the CLI, but you could pass in a binary string or a path to a piece of json with key material. I guess the issue here is how to go from CLI command option to 32-byte `Uint8Array`. Definitely need suggestions on this one.

Reply:
1. I'm assuming we're moving to using `seed` because this gets us ready for a passwordless internet in which authentication is done using wallets.
2. the seed also assumes that the person using the CLI has access to the maintainer key's seed.
3. because this might not be the case, we might want to allow for passing in a zcap for a `bedrock-web-kms` key.
4. In this pattern, the CLI would take an option `--keyConfing` alias `-k` that takes in a piece of json
5. That json document would need to contain the seed for the invocationKey and possibly a zcap with permission to sign as the maintainer.

Exampke `keyConfig.json`
```js
zcap: {
  ...skipped
},
invocationSigner: {
  "type": "Ed25519VerificationKey2020",
  "publicKeyMultibase": "zpublicKeyMultibase",
  "privateKeyMultibase": "zprivateKeyMultibase"
}
```

Reply:
We pass seeds using base58-encoded strings elsewhere, IIRC. @gannan08? There should be examples in things like bedrock-app-identity.

Reply:
looks like the decoding happens here: https://github.com/digitalbazaar/bedrock-app-identity/blob/05a3b09cf55856f86188ee02cde01287a9221bc1/lib/main.js#L119-L134 

So I'll use `bnid` ASAP.

Reply:
Two commits here address this:

https://github.com/digitalbazaar/bedrock-ledger-maintainer-cli/pull/2/commits/d48a60c1fca64991ff2a2eda06a5eaae82317616

https://github.com/digitalbazaar/bedrock-ledger-maintainer-cli/pull/2/commits/7b035cbd5eaec723412ad53b6d018948e50bfc56


Those commits were tested on v1 and a generic bedrock ledger node.
Worked on both with fresh starts.